src/sentry/identity/oauth2.py

```diff
@@ -67,7 +67,7 @@ def get_oauth_access_token_url(self):
         return self._get_oauth_parameter('access_token_url')
 
     def get_oauth_refresh_token_url(self):
-        raise NotImplementedError
+        return self._get_oauth_parameter('access_token_url')
```

Comment:
Not a issue with this PR or anything, but it made me think of this – we should stop writing `get_*` functions and just use properties. We have a ton that are returning data, and have no logic. It'd be nice for the object's interface to reflect that, imo.

Reply:
I don't disagree.